docs/UI/AddSubMenu/assets/AddSubMenuWebEngineApp.txt

```diff
@@ -0,0 +1,17 @@
+participant LocalWebApp
+participant SDL
+participant HMI
+note over LocalWebApp, HMI:<align:center>**Preconditions:**\n1. Aplication (with `appID`) is active on HMI, `AppHMIType` is `WEB_VIEW`\n2. Aplication active template is `WEB_VIEW`\n3. HMI shows app/template title on template layout
+note over HMI:<align:center> Display Aplication web page view\nin the main window
+note over LocalWebApp:<align:center> MENU button\n is displayed
+LocalWebApp->SDL:AddSubMenu (appID, params)
+SDL->HMI:UI.AddSubMenu (appID, params)
+note over HMI:<align:center>Store received data\nassociating it with `appID`
+HMI-->SDL:UI.AddSubMenu (SUCCESS)
+SDL-->LocalWebApp:AddSubMenu (SUCCESS)
+note over LocalWebApp:User taps MENU button
```

Comment:
The diagram contains a show app menu response but no request except this note. Would you mind adding the arrow for `ShowAppMenu` request?

Reply:
@kshala-ford, thank you, please, see fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/219/commits/efdae1bf4ec782ce807b778c03dd6b8d9f7ace48